Minor changes to webhooks docs (#1052)
codatio/codat-docs: docs/using-the-api/webhooks/overview.md

We refer to the HTTP endpoints that you configure to subscribe to Codat's events as _webhook consumers_. A `200` response from the consumer indicates that the webhook message was successfully received. 

With our webhook service, you can leverage the following benefits:

- Automatic retries of failed webhook deliveries according to our [retry schedule](/using-the-api/webhooks/troubleshooting#retry-policy)
- Detailed event logging with a full overview of delivery attempts and payloads
- Make your solution robust with easy event replay to reprocess past events or retry failed events
- Mock event functionality to simplify your testing and development

:::note Using our legacy webhooks? 

> If you have been using our webhooks prior to March 6, 2024, we recommend migrating over to the new service.

Reviewer: ```suggestion
If you have been using our webhooks before March 6, 2024, we recommend migrating them to the new service.
```

> Details on how do migrate can be found in our [migration guide](/using-the-api/webhooks/migration-guide).

Reviewer: ```suggestion
See how you can migrate in our [migration guide](/using-the-api/webhooks/migration-guide).
```

> You can find the old documentation here: 

Reviewer: ```suggestion
You can view the old webhook service documentation here: 
```
